test/functionalTest/cases/1316_create_subscription/post_subs_errors.test

```diff
@@ -343,16 +343,12 @@ Date: REGEX(.*)
 
 08. POST /v2/subscriptions (missing subject condition)
 ======================================================
-HTTP/1.1 400 Bad Request
-Content-Length: 69
-Content-Type: application/json
+HTTP/1.1 201 Created
+Content-Length: 0
+Location: /v2/subscriptions/REGEX([0-9a-f]{24})
 Fiware-Correlator: REGEX([0-9a-f\-]{36})
```

Comment:
Step 8 is no longer valid "as is" as a valid error case for post_subs_errors.test. However, it would be easy to replace this with the following cases:

8a. conditions: {} -> conditions is there, but empty object
8b. conditions: { expression: { } } -> expression is there, but empty object
8c. conditions: { expression: { "q": ""} } -> expression ok, but q is empty (and similar 8d, 8f, etc. with the rest of sub-field that could be included in expression)